client/cli/go/cmds/topology.go

```diff
@@ -126,6 +126,10 @@ var topologyLoadCommand = &cobra.Command{
 
 		// Load current topolgy
 		heketiTopology, err := heketi.TopologyInfo()
+		if err != nil {
+			fmt.Fprintf(stdout, "Unable to get topology information\n")
```

Comment:
No need to print out.  This will happen automatically when the error is returned.